### ISSUE TITLE
tooltip not working with button groups

### ISSUE THREAD

**Issue by nicolaslefresne** [livewire/flux#161]
Hi,

it seems tooltips mess with button groups display:

```
<flux:button.group>
    <flux:button icon="pencil-square" size="sm" tooltip="Edit"></flux:button>
    <flux:button icon="plus" size="sm" tooltip="Add"></flux:button>
    <flux:button variant="danger" icon="trash" size="sm" tooltip="Delete"></flux:button>
</flux:button.group>
```

**Comment by christianheidorn** [comment]
As far as I can tell, it's not just caused by tooltips but also if you only change the size. Can you confirm this @nicolaslefresne ?

**Comment by joshhanley** [comment]
@nicolaslefresne thanks for reporting! Please provide screenshots or comparisons in future reports to make it easier to understand what is going on 🙂

To add some more detail to this report, I tested with the following code and you can see in the screenshot below, that the top button group renders happily but in the bottom button group the buttons all have radiuses and the trash has a dark shadow on the right.

```blade
{{-- Without tooltip --}}
<flux:button.group>
    <flux:button icon="pencil-square" size="sm"></flux:button>
    <flux:button icon="plus" size="sm"></flux:button>
    <flux:button variant="danger" icon="trash" size="sm"></flux:button>
</flux:button.group>

{{-- With tooltip --}}
<flux:button.group>
    <flux:button icon="pencil-square" size="sm" tooltip="Edit"></flux:button>
    <flux:button icon="plus" size="sm" tooltip="Add"></flux:button>
    <flux:button variant="danger" icon="trash" size="sm" tooltip="Delete"></flux:button>
</flux:button.group>
```

![Image](https://github.com/user-attachments/assets/7e7fd3f0-612c-46fc-9b77-e5cd9e8cd5fd)

**Comment by nicolaslefresne** [comment]
Thank you and sorry @joshhanley will do better next time :)

@christianheidorn I tested several cases and the size does not seem to affect the behaviour, only the tooltip :

**Normal version :**
```
<flux:button.group>
    <flux:button icon="pencil-square" tooltip="Modifier"></flux:button>
    <flux:button icon="key"></flux:button>
    <flux:button variant="danger" icon="trash"></flux:button>
</flux:button.group>
```
![Image](https://github.com/user-attachments/assets/2ec9716f-4f41-4458-8b44-c5461db247b4)

**Small version :**
```
<flux:button.group>
    <flux:button icon="pencil-square" size="sm" tooltip="Modifier"></flux:button>
    <flux:button icon="key" size="sm"></flux:button>
    <flux:button variant="danger" icon="trash" size="sm"></flux:button>
</flux:button.group>
```
![Image](https://github.com/user-attachments/assets/baf2a69f-5d9e-4599-917b-6abbacb49abb)

So in short, it seems that adding the tooltip param prevents the button in a button group from having "fluid" borders.

**Comment by christianheidorn** [comment]
@nicolaslefresne Yes, you're right. That was my mistake. I got confused with another issue I was facing with these buttons.

**Comment by calebporzio** [comment]
Thank you for the report, however we have been unable to reproduce this given the information available. Please re-submit this issue with a code snippet we can copy and paste into our own apps to reproduce. Then, we will be happy to help.
